genshin/models/genshin/chronicle/activities.py

```diff
@@ -43,6 +44,78 @@ class Activity(OldActivity[ModelT]):
     total_times: int = 1
 
 
+class DateTime(APIModel):
+    """DateTime model."""
+
+    year: int
+    month: int
+    day: int
+    hour: int
+    minute: int
+
+
+# ---------------------------------------------------------
+# Summer：
+
+
+class SummerStoryRecord(APIModel):
+    """Summer story record."""
+
+    finish_time: typing.Optional[DateTime] = Aliased("finish_time")
+    finished: bool
+    icon: str
+    name: str
+
+
+class SummerStory(APIModel):
+    """Summer story."""
+
+    records: typing.Sequence[SummerStoryRecord] = Aliased("records")
+
+
+class SummerSailingRecord(APIModel):
+    """Summer sailing record."""
+
+    cost_time: int
+    finished: bool
+    id: int
+
+
+class SummerSailing(APIModel):
+    """Summer Sailing."""
+
+    records: typing.Sequence[SummerSailingRecord] = Aliased("records")
+
+
+class SummerChallengeRecord(APIModel):
+    """Summer challenge record."""
+
+    id: int
+    finish_time: typing.Optional[DateTime] = Aliased("finish_time")
+    finished: bool
+    success_num: int
+    skill_use_num: int
```

Comment:
these should be aliased to how they appear in the hoyolab UI. 

Affixes such as `is_foo` or `foo_num` are not needed since annotations are already used to signify the type.